### PR TITLE
docs: changed nim language and added instructions to obtaining api key

### PR DESCRIPTION
**Pull request by dglogo**
# Description

Updated to NIM language and added instructions to obtaining api key from API Catalog

### CODE REVIEW

File: docs/docs/examples/llm/nvidia_nim.ipynb [hunk 1]
```diff
@@ -4,16 +4,20 @@
    "cell_type": "markdown",
```
Comment (nerdai) [inline docs/docs/examples/llm/nvidia_nim.ipynb:4]: <p>I think we should commit a notebook with cell outputs as we had before.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (nerdai) [inline docs/docs/examples/llm/nvidia_nim.ipynb:4]: <p>Looks like this information is repeated?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Review by nerdai — COMMENTED** [review]
Thanks @dglogo for the added details in the notebook. Left a couple of comments in my review.

**Review by logan-markewich — APPROVED** [review]

**Comment by dglogo** [comment]
Thanks @logan-markewich

**Pull request merged**